README.md

```diff
@@ -73,6 +79,8 @@ The PX4 Dev Team syncs up on a [weekly dev call](https://dev.px4.io/en/contribut
   * [Airmind MindPX / MindRacer](https://github.com/PX4/Firmware/labels/mindpx) - [Henry Zhang](https://github.com/iZhangHui)
   * RTPS/ROS2 Interface - [Vicente Monge](https://github.com/vicenteeprosima)
 
+See also [About Us](http://px4.io/about-us/#development_team) (px4.io) and the [contributors list](https://github.com/PX4/Firmware/graphs/contributors) (Github).
+
 ## Supported Hardware
 
 This repository contains code supporting these boards:
```

Comment:
@dagar @bkueng Do you think it makes sense to migrate this flat list "as is" to the user guide as a new section here: https://docs.px4.io/en/flight_controller/ - then cross link back from the readme.

I kind of like the idea because it would be good if the list was only in one place. Also because the user guide does not have a flat list like this, so you need to expand the tree to find the information. Thoughts?

Reply:
Sounds good

Reply:
The only possible issue is "This repository contains code supporting these boards:". So we're actually talking about a level of support here that we test. While in the docs we're talking more about "these boards are supposed to work". So while I definitely need a flat list in the docs, I'm not sure that moving this README list  is "wise".